src/curl.c

```diff
@@ -105,6 +128,11 @@ static int check_connection(const char *test_capath)
 		return -1;
 	}
 
+	curl_ret = swupd_curl_set_timeouts(curl);
```

Comment:
I was reviewing curl.c source code and I think it's better to call swupd_curl_set_basic_options()  on check_connection to use all other configurations for curl as any other connection

Reply:
_swupd_curl_set_basic_options()_ receives a url as parameter. Since this is part of the swupd init I can use version_url or content_url.